<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUser</class>
 <widget class="QWidget" name="EditUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>1500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_2{background: #FFFFFF;
border-radius: 8px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>300</width>
      <height>565</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 300px;
height: 565px;
background: #FFFFFF;
border-radius: 8px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="portrait_Label">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>123</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_LineEdit">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>283</y>
       <width>230</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333333;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>请输入姓名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_LineEdit">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>353</y>
       <width>230</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333333;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>请输入ID号</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="height_LineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333333;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>116</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户年龄：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="age_LineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>105</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333333;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>181</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户身高：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="weight_LineEdit">
    <property name="geometry">
     <rect>
      <x>1106</x>
      <y>170</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333333;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>181</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户体重：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>116</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户性别：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="female_RadioButton">
    <property name="geometry">
     <rect>
      <x>1237</x>
      <y>115</y>
      <width>65</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>女</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="male_RadioButton">
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>115</y>
      <width>65</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>男</string>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rightSide_Btn">
    <property name="geometry">
     <rect>
      <x>962</x>
      <y>300</y>
      <width>80</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 80px;
height: 44px;
background: #FFFFFF;
border-radius: 4px;
border: 2px solid #333333;</string>
    </property>
    <property name="text">
     <string>右侧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftSide_Btn">
    <property name="geometry">
     <rect>
      <x>852</x>
      <y>300</y>
      <width>80</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 80px;
height: 44px;
background: #FFFFFF;
border-radius: 4px;
border: 2px solid #333333;</string>
    </property>
    <property name="text">
     <string>左侧</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>251</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户信息：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userInfo_LineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>240</y>
      <width>680</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333333;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>309</y>
      <width>182</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 182px;
height: 26px;
font-size: 26px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #666666;
line-height: 39px;</string>
    </property>
    <property name="text">
     <string>下肢功能障碍：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_LineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>680</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333333;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>376</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户电话：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>441</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>家庭住址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userTel_LineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>365</y>
      <width>680</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333333;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remark_LineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>680</width>
      <height>135</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333333;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>501</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户备注：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_Btn">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>700</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0D9DDB;
border-radius: 8px;
color:white;
font: 26pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_Btn">
    <property name="geometry">
     <rect>
      <x>1279</x>
      <y>700</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0D9DDB;
border-radius: 8px;
color:white;
font: 26pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modify_Btn">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>700</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0D9DDB;
border-radius: 8px;
color:white;
font: 26pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_Btn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>700</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #F78759;
border-radius: 8px;
color:white;</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{background: #0D9DDB;
border-top-left-radius: 8px;border-top-right-radius:8px}</string>
    </property>
    <widget class="QPushButton" name="close_Btn">
     <property name="geometry">
      <rect>
       <x>1430</x>
       <y>20</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/close.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="title_Label">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>411</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>岁</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>173</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>175</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>kg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
